.github/workflows/ci_workflows.yml

```diff
@@ -9,45 +9,27 @@ on:
 
 jobs:
   tests:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: ubuntu-latest
-            python-version: 3.7
-            toxenv: py37-test-pytest46
-          - os: windows-latest
-            python-version: 3.7
-            toxenv: py37-test-pytest50
-          - os: macos-latest
-            python-version: 3.8
-            toxenv: py38-test-pytest52
-          - os: ubuntu-latest
-            python-version: 3.8
-            toxenv: py38-test-pytest53
-          - os: windows-latest
-            python-version: 3.9
-            toxenv: py39-test-pytest60
-          - os: macos-latest
-            python-version: 3.9
-            toxenv: py39-test-pytest61
-          - os: ubuntu-latest
-            python-version: '3.10'
-            toxenv: py310-test-pytest62
-          - os: ubuntu-latest
-            python-version: '3.10'
-            toxenv: py310-test-pytestdev
-
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install tox
-      run: python -m pip install tox
-    - name: Run tox
-      run: tox -v -e ${{ matrix.toxenv }}
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
```

Comment:
I don't necessarily see why relying on one more layer makes sense here as all the CI setup is super basic for the plugin.

But if you want to, sure.

Reply:
It's a very well maintained middle layer though and does save quite a few lines. I would have a preference to use it if we can but happy to revert if you feel strongly about it.

Reply:
No strong feelings about it here, but I wouldn't want it for doctestplus and remotedata.